src/test/java/nl/dtls/fairdatapoint/service/impl/FairSearchClientImplTest.java

```diff
@@ -51,8 +51,8 @@
  */
 public class FairSearchClientImplTest {
 
-    private static final Logger LOGGER = LogManager.getLogger(FairSearchClientImplTest.
-            class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            FairSearchClientImplTest.class.getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)